lib/swaps/Swaps.ts

```diff
@@ -712,23 +703,27 @@ class Swaps extends EventEmitter {
    * @returns `true` if the resolve request is valid, `false` otherwise
    */
   private validateResolveRequest = (deal: SwapDeal, resolveRequest: ResolveRequest)  => {
-    const { amount } = resolveRequest;
+    const { amount, tokenAddress, cltvDelta } = resolveRequest;
```

Comment:
note the cltvDelta is block number here

You need to expiration here

Reply:
Currently `cltvDelta` here is being set to `reveal_timeout`, are you saying that it needs to be `expiration` instead? The raiden code and docs makes it seem like `expiration` is the block number and `reveal_timeout` is the delta, but I may be wrong. Again I can clarify this with some debugging

Reply:
`reveal_timout` is the lower bound, not the delta, see https://gitter.im/exchangeunion/raiden-swaps?at=5d3ef4fab2f4075cb821d728